diffpy/pdfmorph/pdfmorph_api.py

```diff
@@ -0,0 +1,200 @@
+        minimizing targets during morphing. Default to False.
+    fixed_operations : list, optional
+        A list of string specifying operations will be keep fixed during
+        morphing. Default is None.
+    refine : bool, optional
+        Option to execute the minimization step in morphing. If False,
+        the morphing will be applied with parameter values specified in
+        `morph_config`. Default to True.
+    verbose : bool, optional
+        Option to print full result after morph. Default to True.
+    kwargs : dict, optional
+        A dictionary with morph parameters as keys and initial
+        values of morph parameters as values. Currently supported morph
+        parparameters are:
+
+            - 'scale'
+            - 'stretch'
+            - 'smear'
+            - 'baselineslope'
+            - 'qdamp'
+
+    Returns
+    -------
+    chain : diffpy.pdfmorph.morphs.morphchain.MorphChain
+        instance of morph chain. ``xobj, yobj, xref, yref = chain.xyallout``
+        will return morphed results
+    rv_cfg : dict
+        dictionary of refined parameters
+    rw : float
+        normal rw value
+    pcc : float
+        pearson correlation coefficient
+
+    Examples
+    --------
+    # morphing (xobj, yobj) pair to (xref, yref) pair with scaling
+    from diffpy.pdfmorph import pdfmorph, default_config, plot_morph
+
+    morph_cfg = dict(default_config)
+    morph_cfg['scale'] = 1.01
+    rv = pdfmorph(xobj, yobj, xref, yref, **morph_cfg)
+    morph_chain, morphed_cfg, rw, pearson = rv
+
+    plot_morph(morph_chain)
+    print(morphed_cfg, rw)
+    ..
+    """
+    operation_dict = {}
+    refpars = []
+    # base config
+    rv_cfg = dict(default_config)
+    if kwargs:
+        rv_cfg.update(kwargs)
+    # configure morph operations
+    active_morphs = [k for k, v in rv_cfg.items() if (v is not None) and k in
+                     morph_step_dict]
+    rv_cfg['rmin'] = rmin
+    rv_cfg['rmax'] = rmax
+    rv_cfg['rstep'] = rstep
+    # configure smear, guess baselineslope when it is not provided
+    if (rv_cfg.get('smear') is not None
+        and rv_cfg.get('baselineslope') is None):
+        rv_cfg['baselineslope'] = -0.5
+    # config dict defines initial guess of parameters
+    chain = morphs.MorphChain(rv_cfg)
+    # rgrid
+    chain.append(morphs.MorphRGrid())
+    # configure morph chain
+    for k in active_morphs:
+        morph_cls = morph_step_dict[k]
+        if k == 'smear':
+            [chain.append(el()) for el in morph_cls]
+            refpars.append('baselineslope')
+        else:
+            chain.append(morph_cls())
+        refpars.append(k)
+    # exclude fixed options
+    if fixed_operations:
+        if not isinstance(fixed_operations, Iterable):
+            fixed_operations = [fixed_operations]
+        for opt in fixed_operations:
+            refpars.remove(opt)
+        print("== INFO: Following morphing steps are fixed ==:\n{}\n"
```

Comment:
please put this after `if verbose` or so.  It is better if function does not produce debugging output unless requested.